docs/source/jsonpath.rst

```diff
@@ -78,6 +78,42 @@ or::
 
 Examples like this can be found in one of gabbi's `own tests`_.
 
+When reading from disk you can apply the same JSONPath by adding a ':' to the
+end of your file name. This allows you to store multiple api responses into
+a json file to reduce file management when constructing your tests
```

Comment:
Need a '.' on the end of the sentence?

Also I think it would be wise if we had a small warning here about the impact that putting results in a different file that has to be parsed may have on the readability of the tests. Just a light warning, not something that would dissuade people from doing it, but just to make them aware of the cost/benefit balance.